public/app/core/components/AppChrome/MegaMenu/utils.ts

```diff
@@ -62,57 +58,64 @@ export const hasChildMatch = (itemToCheck: NavModelItem, searchItem?: NavModelIt
   );
 };
 
-const stripQueryParams = (url?: string) => {
-  return url?.split('?')[0] ?? '';
-};
+/**
```

Comment:
I still had to do some special-casing here. I looked at the dashboard page code a little bit to see if I could easily set the nav item to be a "starred" one, but I didn't fancy making changes in somewhere so critical in _this_ PR - unless someone is able to point me in a direction to quickly fix that?

I was getting a dashboard page as only ever having a nav item with ID of `dashboards/browse`